go/metadata/merger_test.sh

```diff
@@ -16,11 +16,43 @@
 #
 ################################################################################
 #
+set +e
 printenv
 
+error=0
+
 $TEST_SRCDIR/$TEST_WORKSPACE/go/metadata/merger --output $TEST_TMPDIR/out.json \
 	$TEST_SRCDIR/$TEST_WORKSPACE/go/metadata/testdata/chrome-linux.json \
 	$TEST_SRCDIR/$TEST_WORKSPACE/go/metadata/testdata/android-browser-gingerbread-nexus-s.json
 
 diff $TEST_TMPDIR/out.json \
-	$TEST_SRCDIR/$TEST_WORKSPACE/go/metadata/testdata/merger-result.json
\ No newline at end of file
+	$TEST_SRCDIR/$TEST_WORKSPACE/go/metadata/testdata/merger-result.json
+if [ $? != 0 ]; then
```

Comment:
Prefer [[ to [.
https://google.github.io/styleguide/shell.xml#Test,_%5B_and_%5B%5B